src/index.html

```diff
@@ -15,13 +15,438 @@
       href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
       rel="stylesheet"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
```

Comment:
The link to the Google Fonts API includes both 'Poppins' and 'Roboto'. Since 'Roboto' is already included in the previous link, this might be redundant. Consider removing one of the links if both fonts are not needed.